### ISSUE TITLE
Naming conflicts?

### ISSUE THREAD

**Issue by ValdarT** [alan-turing-institute/MLJModels.jl#8]
I thought about adding support vector machines through [LIBSVM.jl](https://github.com/mpastell/LIBSVM.jl) but these are now implemented through ScikitLearn so question arises how to handle this kind of situation. Should we (1) somehow distinguish the different implementations by name, or (2) select the most appropriate implementation in each case (so no conflicts because we don't allow 2 implementation of the same model) or (3) not worry about it at all since the user loads whatever libraries they want.

The last option seems problematic if some library supports a lot of models (like ScikitLearn). My first thought would be to distinguish ScikitLearn models by name, for example with 'sk' prefix so SVMClassifier would become skSVMClassifier or something like that but this is not a general solution, of course (e.g., what to do when a pure-Julia SVM implementation becomes available).

**Comment by ablaom** [comment]
It would be great to have Julia native SVM implementation. The ScikitLearn wraps are a temporary expediency. 

This is issue of name duplication has been anticipated. At the moment there is no problem with models in different packages having the same name. Models are identified by (pkg, model) pair. In the present design the only limitation is that you cannot *load* two models into your workspace if they have the same model name. To remove this limitation would be some hassle, but I don't personally see a need to.

Basically, your model implementation can live anywhere, so long as you take these steps so MLJ's `@load` macro can find it:

(i) In your implementation code define appropriate values for the model "traits" `package_name` and `load_path`. See MLJModels/src/DecisionTreeRegressor for an example.

(ii) Raise an issue at MLJRegistry saying you want the relevant package registered (eg, 
[LIBSVM.jl](https://github.com/mpastell/LIBSVM.jl)).
